sunpy/sun/models.py

```diff
@@ -20,6 +20,7 @@
 
 import pandas
```

Comment:
Do we still need to import pandas?

Reply:
No we don't. My mistake, I overlooked it.